src/migrations/201602231400_initial.js

```diff
@@ -17,7 +17,7 @@ function createAccountsTable (knex) {
 function createFulfillmentsTable (knex) {
   return knex.schema.createTableIfNotExists('fulfillments', (table) => {
     table.increments()
-    table.uuid('transfer_id').unique().index()
+    table.uuid('transfer_id').index()
```

Comment:
I tried running with the `unique` command here, but got failures: `Knex:warning - migrations failed with error: alter table "transfers" add primary key ("id") - ORA-02260: table can have only one primary key`. This can be observed locally, and on circleci. FWIW, after removing `unique` from the column and re-running migrations, I saw the same error. It seems that once the database gets in a certain state, the migrations can't execute. The workaround is to re-create the db from scratch.